src/components/shared/Extensions/NavLink/NavLink.tsx

```diff
@@ -21,7 +22,12 @@ const NavLink: FC<PropsWithChildren<NavLinkProps>> = ({
   const titleText = typeof title === 'string' ? title : title && formatMessage(title, titleValues);
 
   return (
-    <NavLinkComponent to={to} title={titleText} {...props}>
+    <NavLinkComponent
+      className={`${className} transition-all duration-normal hover:text-blue-400`}
```

Comment:
Maybe better use clsx instead of `{${className}`? => `clsx(className, "...")`

Reply:
Done